include/swift/AST/DiagnosticEngine.h

```diff
@@ -685,9 +688,8 @@ namespace swift {
     /// but rather stored until all transactions complete.
     llvm::StringSet<llvm::BumpPtrAllocator &> TransactionStrings;
 
-    /// Diagnostic producer to handle the logic behind retriving a diagnostic message,
-    /// from YAML files.
```

Comment:
@xedin I updated this comment to be more general (removed "from YAML files") because in the future we might `emplace` `localization` with a serialized version or with YAML like now. What do you think?

Reply:
Sounds good!